proposals/0029-cooperative-vector.md

```diff
@@ -102,16 +104,23 @@ void ps_main(args) // args: texture, normal, position
 
     // layer0 = inputVector*inputMatrix + biasVector0
     // The matrix and bias are loaded from memory at offsets : moffset0 and boffset0
-    vector<uint32_t, K> layer0 = MatrixVectorMulAdd(inputVector, inputMatrix0, moffset0, biasVector0, boffset0);
+    MatrixRef<DATA_TYPE_UINT32, K, M, MATRIX_LAYOUT_MUL_OPTIMAL> M0 = { inputMatrix0, moffset0, 0 }; 
```

Comment:
Separately, moffset0 isn't defined yet. Would we expect it to be passed as an argument to ps_main? Or perhaps it would be read from some resource?

Reply:
This was missing from the original sample that's been converted to the new API - I think we have to assume that this just isn't shown in this example.